tests/keras/layers/cudnn_recurrent_test.py

```diff
@@ -122,46 +122,45 @@ def test_cudnn_rnn_canonical_to_params_gru():
 
 
 @keras_test
+@pytest.mark.parametrize("rnn_type", ['lstm', 'gru'], ids=['LSTM', 'GRU'])
 @pytest.mark.skipif((keras.backend.backend() != 'tensorflow'),
                     reason='Requires TensorFlow backend')
 @pytest.mark.skipif(not keras.backend.tensorflow_backend._get_available_gpus(),
                     reason='Requires GPU')
-def test_cudnn_rnn_timing():
-    input_size = 1000
-    timesteps = 60
-    units = 256
-    num_samples = 10000
+def test_cudnn_rnn_timing(rnn_type):
+    input_size = 30
+    timesteps = 50
+    units = 128
+    num_samples = 640
```

Comment:
I would recommend a greater number of units, timesteps, and samples (like the original values). This test does not run on CI so it does not need to be fast. Think of it as a benchmark, not a unit test.